common/utils/src/test/scala/org/apache/spark/util/StructuredLoggingSuite.scala

```diff
@@ -192,8 +192,8 @@ class StructuredLoggingSuite extends LoggingSuiteBase {
       s"""
         {
           "ts": "<timestamp>",
-          "level": "$level",
           "msg": "This is a log message",
+          "level": "$level",
```

Comment:
Do we need to change the position like this? Is there a way for us to keep the existing location?

Reply:
I don't have a strong opinion on this one. Let me revert the order change.

Reply:
Done